protos/channel_responses.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+/* Following chromium style (e.g. sync_pb) is the usage for the sync protobuf
+*/
+package publishers_pb;
+
+message SocialLinks {
+  string youtube = 1;
+  string twitter = 2;
+  string twitch = 3;
+}
+
+message SiteBannerDetails {
+  string title = 1;
+  string description = 2;
+  string background_url = 3;
+  string logo_url = 4;
+  repeated int32 donation_amounts = 5;
+  SocialLinks social_links = 6;
+}
+
+enum WalletConnectedState {
+  NO_VERIFICATION = 0;
+  UPHOLD_ACCOUNT_NO_KYC = 1;
+  UPHOLD_ACCOUNT_KYC = 2;
+  /* (Albert Wang) These might be used */
+  PAYPAL_ACCOUNT_NO_KYC = 3;
+  PAYPAL_ACCOUNT_KYC = 4;
+}
+
+message ChannelResponse {
+  string channel_identifier = 1;
+  WalletConnectedState wallet_connected_state = 2;
+  string wallet_address = 3;
+  bool publisher_ads_opted_in = 4;
+  SiteBannerDetails site_banner_details= 5;
+}
+
+message ChannelResponses {
+  repeated ChannelResponse channel_response = 1;
```

Comment:
Repeated fields should have a plural name, for better code readability. It would be awkward to have `channel_responses` as the only field in `ChannelResponses` though. Can we do some name changes here?

Perhaps:

```
message ChannelResponse {
  repeated Channel channels = 1;
}
```

Any other ideas?

Reply:
I think it should be 
```
message ChannelResponses {
  repeated ChannelResponse channel_responses = 1;
}
```

Reply:
To avoid ChannelResponses::channel_responses, how about this:

```
message ChannelResponseList {
  repeated ChannelResponse channel_responses = 1;
}
```

Reply:
Fine with me, I'll merge the change.